dali/operators/reader/file_reader_op.cc

```diff
@@ -44,6 +44,15 @@ When traversing subdirectories, the labels are assigned consecutive numbers.)cod
 
 ``stick_to_shard`` and ``random_shuffle`` cannot be used when this argument is set to True.)code",
       false)
+  .AddOptionalArg<vector<string>>("files", R"(A list of file paths to read the data from.
```

Comment:
Nitpick: inconsistent raw string literal delimiter. I know you don't like it, but keeping them consistent makes it easier to change all of them if we decide to do so.
And we should keep the code (at least locally) consistent.

Reply:
LMFIFY